use std::error::Error;

pub fn solve() -> Result<(String, String), Box<dyn Error>> {
    Ok(("A".to_string(), "B".to_string()))
}